packages/cli/src/lib/implementation/skip-plugins.options.ts

```diff
@@ -7,6 +7,7 @@ export const skipPluginsOption: Options = {
   type: 'array',
   default: [],
   coerce: coerceArray,
+  alias: 'P',
```

Comment:
could we get a unit test for this? should be doable in the middleware tests.

Reply:
There were no issues with adding the suggested `config` alias (`-c`). The problem was that it ended up being ignored during the command execution as `-c` conflicted with the Nx CLI option.

As discussed, let's keep the `config` option without an alias for now.